examples/3d-tiles/package.json

```diff
@@ -16,6 +17,8 @@
     "@loaders.gl/3d-tiles": "^1.2.0",
     "@loaders.gl/core": "^1.2.0",
     "@luma.gl/addons": "^7.2.0",
+    "@luma.gl/core": "^7.2.0",
```

Comment:
`v7.3.0-alpha.5`

Reply:
> v7.3.0-alpha.5 

I was trying to match the version that the deck.gl alpha version is using, not sure if pulling in the latest luma will work...